### ISSUE TITLE
Inventory Assistant

### ISSUE THREAD

**Issue by D-Hitman-H** [VoidArkMods/QoLPlus#66]
Can someone tell me why the seeds aren't thrown out even though I marked them on the right?

**Comment by ShakreeElmi** [comment]
i agree to this. even after saving the options in the presets, reopening the window will show the checkboxes being unchecked.

**Comment by DSul3791** [comment]
I am struggling with this as well. The check boxes seem to do nothing. I check them, they uncheck the second I close. I save them into presets and they uncheck the second I close. I check them and save them and it never throws out anything using check boxes. Mostly for seeds right now but it's annoying.
